mvvmlib/build.gradle

```diff
@@ -3,7 +3,7 @@ apply plugin: 'com.android.library'
 apply plugin: 'maven'
 
 
-version = '0.7.6' // Maven Property
+version = '0.7.7' // Maven Property
```

Comment:
Looks like MvvmListViewModel no longer working, should we need to change a major version, e.g. 0.8.0